coderd/database/dbtestutil/db.go

```diff
@@ -137,12 +144,16 @@ func NewDB(t testing.TB, opts ...Option) (database.Store, pubsub.Pubsub) {
 		t.Cleanup(func() {
 			_ = sqlDB.Close()
 		})
+
 		if o.returnSQLDB != nil {
 			o.returnSQLDB(sqlDB)
 		}
 		if o.dumpOnFailure {
 			t.Cleanup(func() { DumpOnFailure(t, connectionURL) })
 		}
+		if o.maxConns > 0 {
```

Comment:
should it do something to unset the limit if you call it with `0`?

Reply:
The unit test code does not currently set a default `MaxConns` that I can tell. If we unset it with `0`, idk what to default it to :thinking:.

I'm sure the package has some default we could revert to.